repository/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -93,4 +93,8 @@
     <include file="db/changelog/db.changelog-rdm-9033.xml"/>
     <include file="db/changelog/db.changelog-rdm-9338.xml"/>
     <include file="db/changelog/db.changelog-rdm-9024.xml"/>
+    <include file="db/changelog/db.changelog-rdm-9031.xml"/>
```

Comment:
it would be great if we could do some cleanup here. In theory we could have 1 liquibase changelog only all the correct changes. 
If this is not too complicated to refactor

Reply:
good point DONE.